tests/examplefiles/arturo/arturo_test.art.output

```diff
@@ -0,0 +1,4999 @@
+'#! home/.arturo/bin' Comment.Hashbang
+'\n'          Text
+
+'\n'          Text
+
+'; this is a comment' Comment.Single
+'\n'          Text
+
+'; this is another comment' Comment.Single
+'\n'          Text
+
+'\n'          Text
+
+';---------------------------------' Comment.Single
+'\n'          Text
+
+'; VARIABLES & VALUES' Comment.Single
+'\n'          Text
+
+';---------------------------------' Comment.Single
+'\n'          Text
+
+'\n'          Text
+
+'; numbers'   Comment.Single
+'\n'          Text
+
+'a1:'         Name.Label
+' '           Text
+'2'           Literal.Number.Integer
+'\n'          Text
+
+'a2:'         Name.Label
+' '           Text
+'3.14'        Literal.Number.Float
+'\n'          Text
+
+'a3:'         Name.Label
+' '           Text
+'to'          Name.Builtin
+' '           Text
+':complex'    Keyword.Type
+' '           Text
+'['           Punctuation
+'1'           Literal.Number.Integer
+' '           Text
+'2.0'         Literal.Number.Float
+']'           Punctuation
+' '           Text
+' '           Text
+' '           Text
+' '           Text
+' '           Text
+'; 1.0+2.0i'  Comment.Single
+'\n'          Text
+
+'\n'          Text
+
+'; strings'   Comment.Single
+'\n'          Text
+
+'c1:'         Name.Label
+' '           Text
+'"'           Literal.String.Double
+'this is a string' Literal.String
+'"'           Literal.String.Double
+'\n'          Text
+
+'c2:'         Name.Label
+' '           Text
+'{'           Literal.String.Single
+'\n'          Literal.String
+
+' '           Literal.String
```

Comment:
There's still a problem here with String contents getting matched char-by-char...

Reply:
- [x] [replace generic Text with better tokens](https://github.com/pygments/pygments/pull/2259/commits/5ab2b42dd95f7511a10a8f328661cb08d9f05eb3)

Reply:
It's happening only with multiline strings. I'll solve it!

Reply:
How it's going now:
![image](https://user-images.githubusercontent.com/78623871/198035464-c3928f80-68be-490e-9337-b38164d208c7.png)

Reply:
LGTM!

Reply:
🙂